test/boost/README.md

```diff
@@ -68,6 +68,42 @@ Boost tests can also be run using test.py - which is a script that provides
 a uniform way to run all tests in scylladb.git - C++ tests, Python tests,
 etc.
 
+## Execution with pytest
+
+To run test with pytest execute 
+```bash
+pytest test/boost
+```
+To execute only one file, provide the path filename
```

Comment:
Perhaps you meant **source** (not _path_) filename?
I would change this sentence to something like:
To execute all tests in one test/boost source file, run pytest with the source file as a parameter: